### ISSUE TITLE
[Feature Request]: Asus AURA Sync in Lightbar

### ISSUE THREAD

**Issue by maicol07** [Paliverse/DualSenseX#98]
**Is your feature request related to a problem? Please describe.**
It would be great if the lightbar can sync with all the Asus Gear with Asus Aura Sync

**Describe the solution you'd like**
A button to enable Aura Sync connection

**Describe alternatives you've considered**

**Additional context**
Asus AURA SDK webiste https://www.asus.com/campaign/aura/us/AURA-ready.php

**Comment by Paliverse** [comment]
I will be looking into this soon!

**Comment by eVenent** [comment]
Good idea, but I think it will be good to implement #83 first, because projects like [Aurora](https://github.com/antonpup/Aurora
) and [RGB.NET](https://github.com/DarthAffe/RGB.NET) are converting different light APIs into one synchronized system. Just we need plug which will turn on light support in DualSenseX.

**Comment by Psytoshgen** [comment]
Maybe roccat aimo? It’s also support alienfx
